python/src/Analytical_doc.i.in

```diff
@@ -178,3 +183,27 @@ Notes
 Performs the research of the design point and creates a
 :class:`~openturns.AnalyticalResult`, the structure result which is
 accessible with the method :meth:`getAnalyticalResult`."
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::Analytical::getLimitStateTolerance
+"Accessor to limitStateTolerance parameter.
+
+Returns
+-------
+limitStateTolerance : float
+    Tolerance on limit state to assess if the found design point can be used 
+    to estimate the failure probability. Default value is defined in :class:`~openturns.ResourceMap` of  class :class:`~openturns.Analytical`. "
```

Comment:
better mention the full key name than just the class

Reply:
fixed